BranchSDK/BNCReferringURLUtility.m

```diff
@@ -25,10 +26,25 @@ - (instancetype)init {
         self.preferenceHelper = [BNCPreferenceHelper sharedInstance];
         self.urlQueryParameters = [self deserializeFromJson:self.preferenceHelper.referringURLQueryParameters];
         [self checkForAndMigrateOldGbraid];
+        
+        NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
```

Comment:
Don't love this design, since it duplicates the session handling elsewhere. However, we can come back to it when we update session handling in general instead of making this more complex now.

Reply:
Maybe add a comment describing that it is session based instead of time window like the others.